cmd/sanitize-prow-jobs/main.go

```diff
@@ -127,18 +149,34 @@ func main() {
 		os.Exit(0)
 	}
 
-	if len(opt.prowJobConfigDir) == 0 {
-		logrus.Fatal("mandatory argument --prow-jobs-dir wasn't set")
+	if err := opt.validate(); err != nil {
+		logrus.WithError(err).Fatal("Invalid options.")
 	}
-	if len(opt.configPath) == 0 {
-		logrus.Fatal("mandatory argument --config-path wasn't set")
+
+	dtEnforcer, err := opt.deprecateTemplates.GetEnforcer()
+	if err != nil {
+		logrus.WithError(err).Fatalf("Failed to load config from %q", opt.configPath)
 	}
 
 	config, err := dispatcher.LoadConfig(opt.configPath)
 	if err != nil {
 		logrus.WithError(err).Fatalf("Failed to load config from %q", opt.configPath)
 	}
-	if err := determinizeJobs(opt.prowJobConfigDir, config); err != nil {
+
+	if err := determinizeJobs(opt.prowJobConfigDir, config, dtEnforcer); err != nil {
 		logrus.WithError(err).Fatal("Failed to determinize")
```

Comment:
It would be better if we combined the errors here and from the new check if enabled rather than fatal out directly, otherwise ppl might need multiple pushes to see all mistakes they made, which is annoying